src/apps/reservation-list/reservation-material/reservation-info.tsx

```diff
@@ -53,58 +55,83 @@ const ReservationInfo: FC<ReservationInfoProps> = ({
   if (state === "readyForPickup") {
     return (
       <ReservationStatus
-        color={colors.success as string}
+        ariaLabel={t("reservationListStatusIconReadyForPickupAriaLabelText")}
+        color={success as string}
         percent={100}
         infoLabel={readyForPickupLabel}
         label={[pickupLibrary, pickupNumber || ""]}
       >
-        <img src={check} alt="" />
-        {t("readyText")}
+        <div className="counter__value" aria-hidden="true">
+          <img src={check} alt="" />
+          <span className="counter__label">
+            {t("reservationListReadyText")}
+          </span>
+        </div>
       </ReservationStatus>
     );
   }
 
   if (state === "reserved" && pickupBranch && numberInQueue && expiryDate) {
-    // todo string interpolation
     const numberInLineLabel =
       numberInQueue === 1
-        ? t("youAreFirstInQueueText")
-        : `${t("youAreNumberInQueueText")} ${numberInQueue - 1}`;
+        ? t("reservationListFirstInQueueText")
+        : t("reservationListNumberInQueueText", {
+            count: numberInQueue - 1,
+            placeholders: { "@count": numberInQueue - 1 }
+          });
 
     return (
       <ReservationStatus
-        color={colors.default as string}
+        ariaLabel={t("reservationListStatusIconQueuedAriaLabelText", {
+          count: numberInQueue,
+          placeholders: { "@count": numberInQueue }
```

Comment:
In the newest version of useText you do not have to specify the count in both count and placeholders. You can skip the placeholder.